core/dbt/contracts/graph/unparsed.py

```diff
@@ -88,23 +88,21 @@ class Docs(dbtClassMixin, Replaceable):
 
 
 @dataclass
-class HasDocs(AdditionalPropertiesMixin, ExtensibleDbtClassMixin, Replaceable):
+class HasColumnProps(AdditionalPropertiesMixin, ExtensibleDbtClassMixin, Replaceable):
```

Comment:
For my own understanding, is this supposed to reflect a Column, but with additional properties? In other words, the new column object would inherit from both `ColumnBase` (assumed name, not sure of the real name) and this `HasColumnProperties` mixin; and the original `ColumnBase` object is very basic with very simple methods/attributes, but this mixin adds additional properties, and methods that use those properties, as a bundle so to speak.

Reply:
No, it doesn't reflect a column with additional properties. It's really just a step on the path to making these mixins consistent. This particular set of fields is used in both nodes and columns.

Reply:
The mixins used for "unparsed" objects have become confusing, and this rename is just meant to be one step in a journey of making them make more sense. Gerda and I have discussed eventually eliminating the mixing approach in favor of concrete objects with minimal inheritance.

This change doesn't make HasDocs a perfectly coherent class in one stroke. It just gives it a name closer to what it is today, in order to make the code more readable. I'm still not sure what this thing is. It's true that it "has docs", but that wasn't a very useful way of thinking about it, since it has accumulated many more properties, and is really more like a column. Confusingly, things that are clearly not columns inherit from it, and they inherit properties that don't make much sense for them. We intend to continue addressing these issues over time.

Reply:
Sorry, I took too long to write my comment. What Gerda said. :D

Reply:
> Gerda and I have discussed eventually eliminating the mixin approach in favor of concrete objects with minimal inheritance.

Sign me up